tasks/upload-sbom-to-atlas/README.md

```diff
@@ -0,0 +1,16 @@
+# upload-sbom-to-atlas
+Tekton task is gathering SBOM data form working directory, converting it to supported version if needed and uploading it to Atlas.
+Supports both CycloneDX and SPDX format.
+
+## Parameters
+
+| Name                      | Description                                                                                                                                                                                                                          | Optional | Default value                                                                 |
+|---------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------|-------------------------------------------------------------------------------|
+| sbomsDir                  | Directory containing SBOM files. The task will search for CycloneDX JSON SBOMs recursively in this directory and upload them all to Atlas. The path is relative to the 'sboms' workspace.                                            | true     | .                                                                             |
```

Comment:
We have a convention of using Yes/No for the Optional column. So please change this to "Yes". This applies to the rest as well.